src/AWS.Deploy.CLI/FallbackCredentialsFactoryFactory.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using Amazon.Runtime;
+
+namespace AWS.Deploy.CLI
+{
+    public class FallbackCredentialsFactoryFactory : IFallbackCredentialsFactoryFactory
+    {
+        public AWSCredentials Create()
```

Comment:
i couldnt return a static class here

Reply:
also im open to any better naming suggestions people have

Reply:
Since you are not creating `FallbackCredentialsFactory` in this factory then it should be turned into an AWS credentials factory. This would also solve the naming issue with this class.